client/package.json

```diff
@@ -173,7 +173,8 @@
     "vite-plugin-svgr": "^3.2.0",
     "vite-tsconfig-paths": "^4.2.0",
     "whatwg-fetch": "^3.6.1",
-    "xml-js": "^1.6.11"
+    "xml-js": "^1.6.11",
```

Comment:
@lilitkarapetyan - can't we remove some mapping related package dependencies?  I'm thinking about `leaflet` and `geoman`.  We'll almost certainly re-introduce them in the future, but removing them now will help double-check that we have deleted all mapping related content.